rclpy/CMakeLists.txt

```diff
@@ -72,10 +72,25 @@ function(configure_python_c_extension_library _library_name)
   )
 endfunction()
 
+add_library(rclpy_common
+  SHARED src/rclpy_common/src/common.c
+)
+target_include_directories(rclpy_common
+  PUBLIC src/rclpy_common/include
+)
+configure_python_c_extension_library(rclpy_common)
```

Comment:
Since `rclpy_common` is a regular library (not a cpython extension) I think the properties set by this function may not be the right thing to do. Maybe CI will pass if this call is replaced with

```cmake
target_link_libraries(rclpy_common
  ${PythonExtra_LIBRARIES}
)
target_include_directories(rclpy_common
  PUBLIC
  ${PythonExtra_INCLUDE_DIRS}
)
install(
  TARGETS rclpy_common
  ARCHIVE DESTINATION lib
  LIBRARY DESTINATION lib
  RUNTIME DESTINATION bin
)
```

Reply:
Thanks, I'll give it a shot.

Reply:
Updated 251175d

Works locally, let's see if CI agrees.